app/models/appliance_comment.rb

```diff
@@ -1,8 +1,25 @@
 class ApplianceComment < ApplicationRecord
   belongs_to :appliance
 
-  enum category: %i[fix note problem]
+  enum category: %i[fix note brake]
```

Comment:
The comment category values are linked to the ApplianceStateMachine transition names. So when we create a comment with brake category the @appliance.brake! will be called. I think we should rename category to action to be more descriprive If we want to improve readability. But it can got to another PR.

Reply:
In that case, the wrong "break" was used. This should be "break" instead of "brake".

Reply:
Sorry, I will fix the typo.

Reply:
I renamed the value, and added som localization.